packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -348,6 +348,21 @@ export const InputPrompt: React.FC<InputPromptProps> = ({
         return;
       }
 
+      if (key.meta && key.delete) {
```

Comment:
I'm not sure we should add is keyboard shortcut. if we do, we should probably add it in combination with supporting backspace as separate from delete. In vscode `alt backspace` deletes a word right and `alt delete` deletes a word right. @dewitt may also have some thoughts as he as done more work than I have on exactly what keyboard shortcuts we should and shouldn't include.